ResidenceManager/ChatMessage.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Firebase
```

Comment:
It looks this line affects nothing. Can we remove it and same lines in the other files?